### DISCUSSION TITLE
Problem with waypoints

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#279] [Q&A] [answered]
Today I had again a problem that I had once in the past. In the pause I wanted to delete some WPs, but it was not possible. The delete option in the context of long press on a WP was not there.
In the WP overview there was only start and destination, even in I had a lot of WPs and SPs. When I long pressed on an SP or WP it showed only the destination.

<img src="https://github.com/devemux86/cruiser/assets/121371629/61df36d4-9897-4534-85a4-2bf43e611845" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/121371629/16bdd8b8-9835-4dd7-9a9e-b1ea56fcab35" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/121371629/168a1058-e3e3-43f7-a395-6dda14457fdf" width="25%"/><img src="https://github.com/devemux86/cruiser/assets/121371629/5479fcce-4928-41c4-abe7-3d65997d5108" width="25%"/>

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

> In the WP overview there was only start and destination

Waypoints cannot be deleted when there are only 2 of them.

> even in I had a lot of WPs and SPs

This is strange, opened the waypoints list and only saw start + end?
And why the start does not show distance / time to the end?

How did you come to that result?
After route planning or an import or something else?
Can you reproduce the steps?

**Reply by DolomitesTom** [reply, marked as answer]
> opened the waypoints list and only saw start + end?

Yes

I can't reproduce now. Suddenly the delete option for WPs was not available any more and all sps and WPs disappeared in the list. When closing the app and reopening, it was available again.

**Reply by devemux86** [reply]
Were you in route planning or inside navigation (with rerouting)?

**Reply by DolomitesTom** [reply]
As far as I can recall I tried it in navigation mode first, then in pausing navigation, because I didn't want to lose the average speed, but also after stopping the navigation, the effect was the same until I closed and reopened the app.

**Reply by devemux86** [reply]
> then in pausing navigation, because I didn't want to lose the average speed

Average speed is maintained while Cruiser is running in all navigation states (on / pause / off).
(until reset it or restart the app)

**Reply by DolomitesTom** [reply]
hopefully it will be soon available as the statistics in the recorded track 😃.
